### PR TITLE
Move and update benchmarks from relate-benchmarks

### PR DESCRIPTION
**Pull request by gregghz**
I've reorganized the build.sbt file to be more readable and make it
easier to add a "bench" task prefix for sbt.

You now can run benchmarks comparing JDBC, Anorm, and Relate by running
`sbt bench:test`.

The relate integration test has been incorporated into the standard test
run and all tests and benchmarks use H2 instead of MySQL.

I had to remove a test because H2 doesn't support returning all column IDs of the rows inserted. It only returns one. I don't expect this to be okay, but I'm interested in comments on that particular test and how I should go about fixing it. Just using MySQL again might be the only way though.

### CODE REVIEW

**Comment by gregghz** [comment]
This provides us with a real benchmarking tool that provides confidence intervals and gives us neat graphs that look like this: http://i.imgur.com/erxfkOn.png (green: relate -- red: anorm -- purple: jdbc)

**Comment by matthew-lucidchart** [comment]
assuming it works, looks good to me

File: src/test/scala/RelateITSpec.scala [hunk 1]
```diff
@@ -57,8 +57,8 @@ trait Db {
           name VARCHAR(50) NOT NULL,
           trainer_id BIGINT,
           PRIMARY KEY(id),
-          KEY trainer_key (trainer_id)
-        ) ENGINE=InnoDB CHARSET=utf8 COLLATE=utf8_general_ci
+          KEY p_oaks_trainer_key (trainer_id)
```
Comment (msiebert) [inline src/test/scala/RelateITSpec.scala]: Thanks for clarifying the name of that key. Can't leave Prof. Oak out!

Comment (gregghz) [inline src/test/scala/RelateITSpec.scala]: @msiebert H2 requires globally unique key names. We already used trainer_key.

Comment (msiebert) [inline src/test/scala/RelateITSpec.scala]: @gregghz Darn, I was just hoping that you were a big fan of Professor Oak...

**Comment by gregghz** [comment]
I added a `regression:test` that only tests the current relate performance so we can track our performance over time.

**Pull request merged**